<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenshotUi</class>
 <widget class="QDockWidget" name="ScreenshotUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>325</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::BottomDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Screenshot</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="drawSettingsLayout" stretch="0,0,0,0">
       <item>
        <widget class="QLabel" name="penLabel">
         <property name="text">
          <string>Pen Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="penBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="drawSettingsSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="drawToolLayout" stretch="1,0,0,0,0,0,0">
       <item>
        <spacer name="drawToolSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="freeButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/usr/share/extcalc/data/free.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lineButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/usr/share/extcalc/data/line.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rectButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/usr/share/extcalc/data/rect.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ellipseButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/usr/share/extcalc/data/circle.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="textButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/usr/share/extcalc/data/text.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rubberButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/usr/share/extcalc/data/rubber.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="undoLayout">
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redoButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="imageSizeLayout">
       <item>
        <spacer name="imageSizeSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="widthLabel">
         <property name="text">
          <string>Image width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="widthBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>512</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="heightBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
         <property name="value">
          <number>512</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="formatLayout">
       <item>
        <widget class="QLabel" name="formatLabel">
         <property name="text">
          <string>File Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="formatBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="pathLayout">
       <item>
        <widget class="QLabel" name="pathLabel">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pathLine"/>
       </item>
       <item>
        <widget class="QPushButton" name="pathButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/usr/share/extcalc/data/open.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="saveSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
